tests/usm/usm_get_pointer_queries.cpp

```diff
@@ -19,8 +19,12 @@ using namespace sycl_cts;
 
 template <sycl::usm::alloc alloc>
 void run_check(const sycl::queue &queue, sycl_cts::util::logger &log) {
-  const auto &device{queue.get_device()};
   const auto &context{queue.get_context()};
+  // According to the SYCL 2020 (rev. 4, $4.8.4. Unified shared memory pointer
+  // queries) the first device in context should be used for alloc::host.
```

Comment:
```suggestion
  // According to the SYCL 2020 (rev. 4, $4.8.4. Unified shared memory pointer
  // queries) 'get_pointer_device' must return the first device in context for
  // the usm::alloc::host kind.
```
Let's be more specific.